code/modules/integrated_electronics/subtypes/power.dm

```diff
@@ -6,8 +6,8 @@
 	desc = "This can wirelessly transmit electricity from an assembly's battery towards a nearby machine."
 	icon_state = "power_transmitter"
 	extended_desc = "This circuit transmits 5 kJ of electricity every time the activator pin is pulsed. The input pin must be \
-	a reference to a machine to send electricity to.  This can be a battery, or anything containing a battery.  The machine can exist \
-	inside the assembly, or adjacent to it.  The power is sourced from the assembly's power cell.  If the target is outside of the assembly, \
+	a reference to a machine to send electricity to. This can be a battery, or anything containing a battery. The machine can exist \
+	inside the assembly, or adjacent to it. The power is sourced from the assembly's power cell. If the target is outside of the assembly, \
```

Comment:
Two spaces after a period is the correct method

Reply:
Damn, okay. I'll get on that, thanks :)